### PR TITLE
Added Italian translation

### PR DESCRIPTION
**Pull request by fcorallini**

### CODE REVIEW

File: advanced_filters/locale/it/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+msgid "Less Than"
+msgstr "Minore di"
+
+#: forms.py:66
+msgid "Greater Than"
+msgstr "Maggiore di"
+
+#: forms.py:67
+msgid "Less Than or Equal To"
+msgstr "Minore o uguale di"
```
Comment (micuffaro) [inline advanced_filters/locale/it/LC_MESSAGES/django.po:69]: Ciao @fcorallini !

What do you think of `Minore o uguale a` for this string?
Same for line 73.

File: advanced_filters/locale/it/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+msgid "One of"
+msgstr "Uno di"
+
+#: forms.py:61
+msgid "DateTime Range"
+msgstr "Intervallo data e ora"
+
+#: forms.py:62
+msgid "Is NULL"
+msgstr "è Nullo"
```
Comment (micuffaro) [inline advanced_filters/locale/it/LC_MESSAGES/django.po:49]: I'd argue that we could keep the values as they are, for example:
`è NULL`
`è TRUE`
`è FALSE`
But I understand why you did that here, it feels like the statement flows better. :)

Comment (fcorallini) [reply advanced_filters/locale/it/LC_MESSAGES/django.po:49]: Hi @micuffaro, I'have taken the spanish translation as guideline. And I prefer the italian version.

File: advanced_filters/locale/it/LC_MESSAGES/django.po (hunk 1 above)
Comment (fcorallini) [reply advanced_filters/locale/it/LC_MESSAGES/django.po:69]: Looks good to me!

Comment (asfaltboy) [reply advanced_filters/locale/it/LC_MESSAGES/django.po:69]: @fcorallini are you going to address the issue ^?

**Review by asfaltboy — COMMENTED** [review]
@fcorallini if you rebase on develop, the CI fails should be gone now.

Please let us know if you'd like to address the last suggestion below, and then LGTM :shipit:

**Review by asfaltboy — CHANGES_REQUESTED** [review]
Looking good, except that 1 small commit got into the branch unintentionally I think.

Let me know if you have any trouble with it, I can push the cleanup as well.

**Review by asfaltboy — APPROVED** [review]
looking good, LGTM 🚢

**Pull request merged**